### ISSUE TITLE
multisite-setup, setup.php `plugins` key does not allow disabling plugins

### ISSUE THREAD

**Issue by drzraf** [getgrav/grav-learn#959]
https://github.com/getgrav/grav-learn/blob/develop/pages/08.advanced/05.multisite-setup/docs.17.md
indicates that `user/setup.php` could do something like:
```php
return [
    'plugins' => []
];
```

I tried:
```php
return [
    'plugins' => [
        'problems' => [
            'enabled' => 'false'
        ],
        'git-sync' => [
            'enabled' => 'false'
        ]
    ],
```

... in order to disable a set of plugins, without success.

Whether or not this is supported, this must be documented.
Note: I couldn't find the necessary codebase references to dig for the underlying logic.

**Comment by mahagr** [comment]
Setup is not meant to do that. It just initializes the system so that everything loads from the right place -- including the configuration. Configuration files that get loaded later will override everything in the setup.

**Comment by drzraf** [comment]
Yeah, but it's documented it could already do that (= configure plugins):

![Screenshot from 2021-12-13 11-29-41](https://user-images.githubusercontent.com/1169926/145830526-34f6dc06-9dad-4c45-a7c0-22362670a6ac.png)

So one would expect it could disable plugins too.

**Comment by mahagr** [comment]
No, even that won't work if you override the keys. Besides above example has been superseded with a better solution anyways.

**Comment by drzraf** [comment]
Could you please remove that misleading chunk from the documentation and put the _better solution_? I'm interested knowing what best practices are advised.
